app/javascript/controllers/flash_controller.js

```diff
@@ -9,13 +11,15 @@ export default class extends Controller {
       setTimeout(() => {
         this.disconnect();
       }, 3500);
-    }, 50);
+    }, 10);
   }
 
   disconnect() {
+    this.element.classList.remove('opacity-100');
+    this.element.classList.add('opacity-0');
+
     setTimeout(() => {
-      this.element.classList.remove('opacity-100');
-      this.element.classList.add('opacity-0');
-    }, 50);
+      this.flashTarget.remove();
```

Comment:
Kanskje heller 

```javascript
      this.flashTarget.classList.add('sr-only');
```

for at Flash-meldingen forblir i DOM for skjermlesere.

Reply:
Sr-only fungerte veldig bra :D